site/docs/master/output-file-format.md

```diff
@@ -57,7 +57,7 @@ rootBucket/
 ```
 Note that this file includes detailed info about your volume snapshots in the `status.volumeBackups` field, which can be helpful if you want to manually check them in your cloud provider GUI.
 
-## file format version: 1
+## File Format Version: 1
```

Comment:
we'll want to bump the version to `1.1.0`.

Reply:
Reviewing the rest, I see there's a new section that now describes the 1.1 format, so actually, I wouldn't expect this section to change at all - in other words, you'll want to undo the changes to the layout here.

Perhaps we can rearrange this page a bit, so something like the following:

- (summary)
- Example backup JSON file
- Output file format versioning
- Versions
    - Version 1.1 (current)
    - Version 1.0